src/main/java/com/jvm_bloggers/frontend/public_area/rss/RssInformationPage.java

```diff
@@ -0,0 +1,72 @@
+package com.jvm_bloggers.frontend.public_area.rss;
+
+import com.jvm_bloggers.frontend.public_area.AbstractFrontendPage;
+import com.jvm_bloggers.frontend.public_area.HomePage;
+
+import org.apache.wicket.markup.html.link.ExternalLink;
+import org.apache.wicket.request.Url;
+import org.apache.wicket.request.cycle.RequestCycle;
+import org.wicketstuff.annotation.mount.MountPath;
+
+import java.util.Arrays;
+
+import static com.jvm_bloggers.core.rss.BlogPostsController.RSS_FEED_MAPPING;
+
+@MountPath("rss")
+public class RssInformationPage extends AbstractFrontendPage {
+    private static final String FEED_URL_PATTERN = "%s%s";
+
+    private final String feedUrl;
+
+    public RssInformationPage() {
+        this.feedUrl = String.format(FEED_URL_PATTERN, getRootUrl(), RSS_FEED_MAPPING);
+
+        add(new ExternalLink("rss_feed_url", feedUrl, feedUrl));
+
+        Arrays.stream(FeedParameter.values())
+            .forEach(e -> add(getExternalLink(e)));
```

Comment:
I am preparing a release, this one is a rather cosmetic change, so if you decide to add it, please create separate PR.